Foundation/NSObjCRuntime.swift

```diff
@@ -404,3 +431,77 @@ public func NSClassFromString(_ aClassName: String) -> AnyClass? {
     
     return _typeByName(aClassNameWithPrefix) as? AnyClass
 }
+
+// The following types have been moved to FoundationNetworking. They exist here only to allow appropriate diagnostics to surface in the compiler.
+
+@available(*, unavailable, message: "This type has moved to the FoundationNetworking module. Import that module to use it.",
+    renamed: "FoundationNetworking.CachedURLResponse")
+public typealias CachedURLResponse = AnyObject
```

Comment:
Note: these should *not* use `renamed:`, because we want people to continue using the unqualified spelling to refer to them once they've imported FoundationNetworking. (That should still work without ambiguity because unavailable declarations should only be used if there's no valid available one.)

Reply:
We're relying on the latter here.

Reply:
In general, I cannot mandate a spelling be qualified or not; it is my job to make _both_ work.

Reply:
(but I agree we shouldn't be recommending to use the qualified one if possible.)

Reply:
Right. I'm specifically talking about `renamed:` because that changes the error message and creates a fix-it.